lib/check/defdelegate_order.ex

```diff
@@ -0,0 +1,71 @@
+defmodule CompassCredoPlugin.Check.DefdelegateOrder do
+  use Credo.Check,
+    base_priority: :normal,
+    category: :readability,
+    explanations: [
+      check: """
+        Prefer placing defdelegate functions the functions.
```

Comment:
```suggestion
        Prefer placing defdelegate before public and private functions.
```

Reply:
Updated in [9853a6e](https://github.com/nimblehq/compass-credo-plugin/pull/10/commits/9853a6ecac8307917297512ca42d711960509399)